helix-term/src/config.rs

```diff
@@ -33,6 +40,12 @@ pub enum ConfigLoadError {
     Error(IOError),
 }
 
+impl Default for ConfigLoadError {
+    fn default() -> Self {
+        ConfigLoadError::Error(IOError::new(std::io::ErrorKind::NotFound, "place holder"))
```

Comment:
Did you mean this to be "place holder"?

Reply:
Yeah this is only used for testing and eben there only acts as a placeholder error for loading local config (all tests use global config). See `Config::load_test`